kratos/utilities/parallel_utilities.cpp

```diff
@@ -84,6 +84,15 @@ int ParallelUtilities::GetNumProcs()
 #endif
 }
 
```

Comment:
yes, as @philbucher pointed out, this one was deliberately left out. We should NOT use this to be forward compatible